errors/errors.go

```diff
@@ -100,3 +101,18 @@ func StackTrace(err error) []string {
 
 	return nil
 }
+
+func Combine(errs []error) error {
+	if len(errs) > 0 {
```

Comment:
Nit: I'd prefer a smaller `if` statement here:

```go
if len(errs) == 0 {
      return nil
}

// ...
```

It'd be nice if we didn't need the `if` statement at all, but I think this is at least a small step forward 👍